### ISSUE TITLE
Bollinger band styling breaks plot

### ISSUE THREAD

**Issue by jraviotta** [freqtrade/freqtrade#2289]
The bb styling in freqtrade/plot/plotting.py does not render as expected.
```python
    if 'bb_lowerband' in data and 'bb_upperband' in data:
        bb_lower = go.Scattergl(
            x=data.date,
            y=data.bb_lowerband,
            name='BB lower',
            line={'color': 'rgba(255,255,255,0)'},
        )
        bb_upper = go.Scattergl(
            x=data.date,
            y=data.bb_upperband,
            name='BB upper',
            fill="tonexty",
            fillcolor="rgba(0,176,246,0.2)",
            line={'color': 'rgba(255,255,255,0)'},
        )
        fig.add_trace(bb_lower, 1, 1)
        fig.add_trace(bb_upper, 1, 1)
```

Given I am calling the following from a jupyter notebook:
```python
chart = generate_candlestick_graph(pair=pair,
                                       data=data,
                                       trades=trades,
                                       indicators1=['bb_upperband', 'bb_lowerband'],
                                       indicators2=[])
```

`chart.show(renderer="browser")` produces:
![Screen Shot 2019-09-22 at 6 58 22 PM](https://user-images.githubusercontent.com/3618365/65395584-01073700-dd6b-11e9-93a5-4d2733d9b303.png)

When the above code is removed, this is produced:

![Screen Shot 2019-09-22 at 6 49 44 PM](https://user-images.githubusercontent.com/3618365/65395554-66a6f380-dd6a-11e9-809b-12427b324081.png)

In addition to the problems rendering, the automatic addition of the indicators is not obvious. I understand offering some defaults for the user, but if duplicates are specified in `indicators1[]`, it is not obvious why the legend contains duplicates without looking at the source.
![Screen Shot 2019-09-22 at 7 02 29 PM](https://user-images.githubusercontent.com/3618365/65395622-90144f00-dd6b-11e9-920c-becf26e32fda.png)

I can create a PR that removes the offending lines if that is helpful.

**Comment by xmatthias** [comment]
The problem with bollinger bands is how they should be rendered, which is as a band, which is not possible by using `indicators1` or `indicators2`.

That's why this is added as "default" if it's available. 
You're right, documentation seems to miss this.

![2019-09-23-061745_1502x759_scrot](https://user-images.githubusercontent.com/5024695/65401637-e4e6b280-ddc9-11e9-8010-4e8914cb3194.png)

Additionally, this method seems to be currently broken in plotly (https://github.com/plotly/plotly.js/issues/2284) for `Scattergl` (the above code was generated by changing the code for this figure to `go.Scatter()`.

So i don't think that adding Bollinger bands is the wrong thing to do (assuming it's working as it should). 

We may should have this documented, and changed `Scattergl` to `Scatter` until that issue in plotly is fixed, but i would not remove the Bollinger code, since that would remove the possibility to show a "band" as shown above.
We could also skip these indicators - but assuming it renders correctly it's just a duplicate indicator ...which the documentation should have told you is automatically added ..

**Comment by jraviotta** [comment]
It seems like the best solution is to use Scatter to produce the intended behavior. While documentation would be helpful, mentioning the auto addition of BB indicator, wouldn't answer the question which led me to search for the root cause. That question was, 'why are my plots broken?' If the indicator renders correctly, then the behavior is obvious. Let me know if you want a PR.

**Comment by xmatthias** [comment]
Absolutely yes.

Would be great if we could also include a reference to the mentioned plotly issue above - so we have some reference to know why this was changed from `Scattergl` to `scatter` without looking through the commit log.

**Comment by xmatthias** [comment]
fixed in  #2297
